### ISSUE TITLE
Documentation help wanted!

### ISSUE THREAD

**Issue by spdegabrielle** [racket/racket#4130]
Hi 👋

We are using this issue to focus on documentation! Documentation is essential for new learners and experienced programmers alike. It helps make our community inclusive by extending a friendly hand to those that are less familiar with Racket. It also helps us find the bugs and issues with the code itself, because we test and try things out as we document.

There are several ways to contribute, you can read more about them on this page: 
https://github.com/racket/racket/wiki/Ways-to-contribute-to-Racket

If you are adding some documentation, please comment below. 

Welcome! We're so glad you're here!

❤️ the Racket community

**Comment by zyrolasting** [comment]
Quick apology, since I think the blog post might be my fault. Will remove it from the index, since I didn't mean it to be there.

**Comment by zyrolasting** [comment]
On that note: The online docs are _in effect_ a public blog that anyone on the Internet can populate. I don't see documentation improving until that stops.

**Comment by samth** [comment]
@zyrolasting I'm confused by the connection you're drawing. Why does the presence of documentation for all packages prevent improvement?

**Comment by zyrolasting** [comment]
> [...] the presence of documentation for all packages [...]

That's not the concern. If anyone from the public can come in and write Scribble wherever they want.  I see how that leads to some of the messiness mentioned in the Discourse thread, such as a dev diary entry of mine appearing in the doc index.

My suggestion would be to treat third-party documentation as untrusted code, because it is. That implies a notion of scoping that isn't happening right now, and I'm unsure which approach would work best there.

**Comment by spdegabrielle** [comment]
Guys! Now I have to close this and do a new one without your discussion 😡😭🤣

**Comment by spdegabrielle** [comment]
And what blog post? @zyrolasting

**Comment by zyrolasting** [comment]
> Guys! Now I have to close this and do a new one without your discussion

This is kind of my point.

> And what blog post? @zyrolasting

https://docs.racket-lang.org/denxi-journal/10_January_2021.html

I need to update my `info.rkt` to remove this document from the index.

**Comment by jryans** [comment]
> That's not the concern. If anyone from the public can come in and write Scribble wherever they want, then I see how that leads to some of the messiness mentioned in the Discourse thread. A dev diary entry of mine appeared in the doc index, for instance.

I think when the Discourse thread mentions "blog post", they are talking about this [contributing tutorial](https://blog.racket-lang.org/2017/09/tutorial-contributing-to-racket.html) on the Racket blog, and not about diary entries that might have slipped onto the packages doc site accidentally. I believe it's a reference to indicate that Racket contribution docs are spread out across several entire sites at the moment.

About the packages doc site generally, personally I find it quite fun and welcoming that anyone can write their package's documentation and have it appear in single shared community resource. It's one of the factors that got me excited about the Racket community and pushed me to investigate it further, since it's quite a unique feature that most language ecosystems don't have.

**Comment by zyrolasting** [comment]
I understand your feelings and identity with them. But I am uncomfortable with how the system works, and am trying to get off of it. I hope that your excitement is couched in empathy for my position.

**Comment by spdegabrielle** [comment]
Yes this issue is meant to be easily found by someone looking to contribute via GitHub
My role will be to support them.
I’ve created a new issue

**Comment by soegaard** [comment]
zyrolasting:
> This is kind of my point re: an editorial process. I'm not sure the docs can use an honor system for the long term.

When it comes to the "official" docs under github.com/racket/ contributions need to be merged before they appear live.

I see your point with respect to the `denxi-journal`, but I am more optimistic. If problems arise,
adjustments can be made to the process.
